### PR TITLE
Update ibm-ctg-connector-release-notes.adoc (Do not merge until release)

### PR DESCRIPTION
**Pull request by alejandrodifiore**
VS 1.1.1: Added support for customize the response time in the IBM CTG Config (Additional Settings section).

### CODE REVIEW

File: modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+|===
+|Software |Version
+|Mule Runtime |3.8
+|IBM CICS TG (for Multiplatforms and for z/OS) |9.1 and 9.2
+|IBM CICS TG SDK |9.1 and 9.2
+|===
+
+[IMPORTANT]
+====
+* The remote CICS TG must be at the same release level or a higher than the version of the JCA resource adapter.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc]: "The remote CICS TG must be at the same release level or a higher than the version of the JCA"
Is there a table showing the correlation between versions? I mean, how does the user know which version of CICS TG is at the same release level as the JCA version? Do they match the numbers?

File: modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+* The JCA resource adapter must match the CICS TG platform. For example, if you have CICS TG for Multiplatforms, you cannot connect to it to or from a CICS TG for z/OS resource adapter.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc]: "...you cannot connect to it **to** or from a..." 
This part is difficult to understand for me, was it meant to be "to it, or from a..." ?

File: modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc (hunk 1 above)
Comment (martinpaoloni) [reply modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc]: I'm not sure. Please consider that this has been part of the release notes for quite a while, and was not written by the connectors team, but by a CTG specialist.
I assume a CTG user understand what this paragraph talks about. If you feel this needs clarification, perhaps @sumitbafna or John Joseph can help.

File: modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc (hunk 2 above)
Comment (martinpaoloni) [reply modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc]: "...you cannot connect *with* it, to or from a CICS TG..." may sound better. Feel free to change this.

File: modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc (hunk 1 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc]: Okay, thanks. Noting here to revisit this statement in the future.

File: modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc (hunk 2 above)
Comment (mcherem) [reply modules/ROOT/pages/connector/ibm-ctg-connector-release-notes.adoc]: Added 'with it' for clarity.

**Pull request merged**